settings.yaml

```diff
@@ -44,6 +44,16 @@ vectorstore:
 qdrant:
   path: local_data/private_gpt/qdrant
 
+sqlmode:
+  enabled: false
+  dialect: mysql
+  driver: pymysql
+  db_host: ${HOSTNAME:localhost}
+  db_user: ${USERNAME:root}
+  db_password: ${PASSWORD:}
+  database: ${DATABASE:}
+  tables: ["${TABLES:}"]
```

Comment:
The env var are specific to SQL/DB world, so I'd personally prefix them with something like `SQL_` or `DB_`.

Additionally, `TABLES` is set to plural, while its usage shows that only a single name is supported, which is misleading.

Reply:
The `tables` variable accepts a list of table names (you can setup multiple tables as context- `SQLTableRetrieverQueryEngine` will select the right table to query).

It's used in the code like- 
`for table_name in self.metadata_obj.tables:
            table_schema_objs.append(SQLTableSchema(table_name=table_name))`

Reply:
@ishaandatta you are right - `tables` does accept a list of string, but the env var `TABLES` seems to only accept one table.

Let me ask this: how would you pass several tables names in the env variable `TABLES`?

Reply:
Right, makes sense- I'm not actually sure how to define an array/list here in the env file.
Figured square brackets were enough as I tested and it works for multiple tables.

I tested it with the following value-
`tables: ["EMPLOYEE", "DEPARTMENT"]`